pynautobot/core/app.py

```diff
@@ -103,6 +118,78 @@ def custom_choices(self):
         ).get()
         return custom_fields
```

Comment:
```suggestion
        return self.custom_fields()
```
You need just that after `logger`.

Reply:
Done ✓